azure-pipelines/secure-development-tools.yml

```diff
@@ -46,15 +39,14 @@ steps:
   condition: ne(variables['DisableAPIScan'], 'true')
 
  ### Run APIScan
-- task: APIScan@1
+- task: APIScan@2
   displayName: Run APIScan
   inputs:
     softwareFolder: $(Agent.TempDirectory)\APIScanFiles
     softwareName: 'StreamJsonRpc'
     softwareVersionNum: '$(Build.BuildId)'
     isLargeApp: false
-    toolVersion: LatestPreRelease
-  continueOnError: true
+    toolVersion: '3.1.29231'
```

Comment:
Where are the versions published? Adding a comment here with a link may help us keep it current over time.

Reply:
I added it